apis/flowcollector/v1beta2/flowcollector_types.go

```diff
@@ -384,6 +386,7 @@ type FlowCollectorEBPF struct {
 	// - `PacketTranslation`: enable enriching flows with packet's translation information. <br>
 	// - `EbpfManager`: allow using eBPF manager to manage netobserv ebpf programs. <br>
 	// IMPORTANT: This feature is available as a Developer Preview.<br>
+	// - `UDNMapping`, to enable interfaces mappind to udn. <br>
```

Comment:
Like above:
```
// IMPORTANT: This feature is available as a Developer Preview.<br>
```

Reply:
Also, like for NetworkEvents we mention `It requires using the OVN-Kubernetes network plugin with the Observability feature.` => we should tell something similar